include/s3select_oper.h

```diff
@@ -217,6 +217,7 @@ class scratch_area
 private:
   std::vector<std::string_view> m_columns{128};
   int m_upper_bound;
+  char m_column_delimiter;
```

Comment:
the scratch area is designed to contain column data of different  input types (CSV, Parquet, JSON and others)
it should not handle output serialization.